### PR TITLE
Use fa gear for default icon

### LINKED ISSUE #52: Change the default app icon to a Font Awesome Gear

### PR DESCRIPTION
**Pull request by ericfranz**
Fixes #52

### CODE REVIEW

**Comment by ericfranz** [comment]
@nickjer

**Comment by ericfranz** [comment]
screen shot 2016-10-24 at 4 12 03 pm

**Comment by ericfranz** [comment]
This is also a step towards supporting Font-Awesome icons as an alternative to icon.png in the root.

**Comment by nickjer** [comment]
I am fine with this.

File: app/views/products/show.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <div class="row">
   <div class="col-md-2 col-sm-2 col-xs-12">
     <p>
-      <%= image_tag app_icon_path(@product.app.name, @product.app.type, @product.app.owner), size: '100x100', title: @product.app.icon_path %>
+      <%= app_icon_tag(product.app) %>
```
Comment (ericfranz) [inline app/views/products/show.html.erb]: Should have used @product here

Comment (ericfranz) [inline app/views/products/show.html.erb]: Fixed.

**Pull request merged**